### DISCUSSION TITLE
2.9.0-rc2 Companion missing ADJUST_GVAR special function

### DISCUSSION THREAD

**Discussion by rburrow87** [EdgeTX/edgetx#3931] [Q&A]
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### What part of EdgeTX is the focus of this bug?

Companion

### Current Behavior

ADJUST_GVAR does not seem to pull into companion, which is supposed to be SF4 below:
![image](https://github.com/EdgeTX/edgetx/assets/1323809/7e160f33-2d77-4be2-a79e-b2ac82bb9263)

In the TX (TX16S MKII):
![screen-2023-08-11-153253](https://github.com/EdgeTX/edgetx/assets/1323809/8277b05e-d23c-47a0-b304-9134eaf2c1a3)

### Expected Behavior

ADJUST_GVAR SF should populate in Companion

### Steps To Reproduce

Create an Adjust special function like in the above screenshot, then read the model in Companion

### Version

2.9.0

### Transmitter

Radiomaster TX16S / TX16SMK2

### Operating System (OS)

Windows

### OS Version

Windows 11 22H2

### Anything else?

Tried my best to search for an existing issue!

**Comment by elecpower** [comment]
@rburrow87 works for me with a new file and the sim. Can you please upload a copy of files you are reading into Companion.

**Comment by elecpower** [comment]
@rburrow87 please check you Companion radio profile settings to ensure option nogvars is not ticked as I can replicate if it is ticked.

**Comment by rburrow87** [comment]
Ahh that's what it was. I thought it might have been the enabled features list causing it to be missing but Global Variables was visible and enabled so I didn't think to check radio profile.

**Comment by elecpower** [comment]
Can never have too many places to check lol
